src/python/pants/backend/python/subsystems/setup.py

```diff
@@ -79,6 +80,34 @@ class PythonSetup(Subsystem):
         advanced=True,
         metavar="<requirement>",
     )
+
+    @memoized_property
+    def interpreter_constraints(self) -> tuple[str, ...]:
+        # TODO: In 2.17.0.dev0 we should set the default above to None and tweak the message here
+        #  appropriately.
+        if not self.options.is_default("interpreter_constraints"):
```

Comment:
The not reads backwards. Is this right?

Reply:
Ooops, that was in there to test something out and I didn't revert it. Hooray for code reviews!